docs/usage/configuration/config_documentation.md

```diff
@@ -533,6 +548,21 @@ listeners:
     bind_addresses: ['::1', '127.0.0.1']
     type: manhole
 ```
+Example configuration #3:
+```yaml
+listeners:
+  # UNIX socket listener: for when Synapse is behind a reverse proxy that can utilise such.
+  #
+  # Note that x_forwarded will default to true, when using a UNIX socket. Please see
+  # https://matrix-org.github.io/synapse/latest/reverse_proxy.html.
+  #
+  - path: /var/run/synapse/main_public.sock
+    mode: 660
```

Comment:
Based off the [other comment](https://github.com/matrix-org/synapse/pull/15708#discussion_r1231518835), it seems like this would give permission errors to people. We should set a more sane default for the example here.

Reply:
That's fair. I can take it off, as less-is-more probably would apply here.

Reply:
Fixed in [9882e0a](https://github.com/matrix-org/synapse/pull/15708/commits/9882e0afdc00cb92f70786ad3807598e230fe93d)